### PR TITLE
Fixes #20232: Git warning about branch name in install logs

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/20232

### CODE REVIEW

File: rudder-webapp/SOURCES/rudder-webapp-postinst [hunk 1]
```diff
@@ -74,12 +74,12 @@ fi
 cd /var/rudder/configuration-repository
 if [ ! -d /var/rudder/configuration-repository/.git ]; then
 
-  git init --shared=group >> ${LOG_FILE}
-
   # Specify default git user name and email (git will refuse to commit without them)
   git config user.name "root user (CLI)"
   git config user.email "root@localhost"
 
+  git init --shared=group >> ${LOG_FILE}
```
Comment (ncharles) [inline rudder-webapp/SOURCES/rudder-webapp-postinst:81]: this will fail with a 
`fatal: not in a git directory

**Review by ncharles — CHANGES_REQUESTED** [review]
you cannot git config user.name if you are not in a git repository

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Review by ncharles — APPROVED** [review]

**Pull request merged**